### PR TITLE
Adding support for Razer DeathAdder 1800

### LINKED ISSUE #883: Support for DeathAdder 1800
**Device Info**

- Name: Razer Death Adder

- Product Number: RZ01-00850100-R3C1

- Model Number: RZ01-0085

**Device Info**
```
Bus 004 Device 002: ID 1532:0038 Razer USA, Ltd 
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 (Defined at Interface level)
  bDeviceSubClass         0 
  bDeviceProtocol         0 
  bMaxPacketSize0         8
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x0038 
  bcdDevice            2.00
  iManufacturer           1 Razer
  iProduct                2 Razer DeathAdder 1800
  iSerial                 0 
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength           84
    bNumInterfaces          3
    bConfigurationValue     1
    iConfiguration          0 
    bmAttributes         0xa0
      (Bus Powered)
      Remote Wakeup
    MaxPower              100mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      2 Mouse
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      94
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 No Subclass
      bInterfaceProtocol      0 None
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     159
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0010  1x 16 bytes
        bInterval               8
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      41
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               8
Device Status:     0x0000
  (Bus Powered)
````
```
leandro@Casa:~$ ls -lah /dev/input/by-id/
total 0
drwxr-xr-x 2 root root 120 jul 25 08:11 .
drwxr-xr-x 4 root root 520 jul 25 08:11 ..
lrwxrwxrwx 1 root root   9 jul 25 08:11 usb-Razer_Razer_DeathAdder_1800-event-mouse -> ../event3
lrwxrwxrwx 1 root root   9 jul 25 08:11 usb-Razer_Razer_DeathAdder_1800-if01-event-kbd -> ../event4
lrwxrwxrwx 1 root root   9 jul 25 08:11 usb-Razer_Razer_DeathAdder_1800-if02-event-kbd -> ../event5
lrwxrwxrwx 1 root root   9 jul 25 08:11 usb-Razer_Razer_DeathAdder_1800-mouse -> ../mouse0
```

**Packet Captures**
I'm not using Windows at all at the moment

**Synapse Screenshots**
I'm not using Windows at all at the moment

### PR DESCRIPTION
**Pull request by nmasnadithya**
Fixes #883

### CODE REVIEW

File: daemon/openrazer_daemon/hardware/mouse.py [hunk 1]
```diff
@@ -1281,3 +1281,23 @@ def _resume_device(self):
...
+               'get_logo_active', 'set_logo_active']
+
+    DPI_MAX = 1800
+
+    DEVICE_IMAGE = "https://rzrwarranty.s3.amazonaws.com/a7daf40ad78c9584a693e310effa956019cdcd081391f93f71a7cd36d3dc577e.png"
+
+    RAZER_URLS = {
+        "top_img": "https://rzrwarranty.s3.amazonaws.com/a7daf40ad78c9584a693e310effa956019cdcd081391f93f71a7cd36d3dc577e.png",
+        "side_img": None,
+        "perspective_img": None
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/mouse.py]: delete the two lines with `None` please

Comment (z3ntu) [reply daemon/openrazer_daemon/hardware/mouse.py]: fixed myself ;)

**Comment by z3ntu** [comment]
Hi, thanks for the pull request! I've rebased your commits on master and squashed them together; so if you want to modify the pull request locally you should run git fetch and reset your worktree with git reset --hard.

Do all effects and dpi work properly? If so, we can merge this :)

**Comment by nmasnadithya** [comment]
Everything works properly. Only issue I've seen is when I open RazerGenie the initial dpi values being weird. But it resolves automatically the moment I try to edit it

File: daemon/openrazer_daemon/hardware/mouse.py (hunk 1 above)
Comment (nmasnadithya) [reply daemon/openrazer_daemon/hardware/mouse.py]: Sorry, I was out

**Comment by leandrobattochio** [comment]
> Everything works properly. Only issue I've seen is when I open RazerGenie the initial dpi values being weird. But it resolves automatically the moment I try to edit it

I'm trying to make it work but no success. Already cloned and builded the package from your branch. Device is detected, DPI values are weird. Cannot edit with Polychromatic. With RazerGenie, the values are weird but they get right after I try to edit. But still, they are not saving or I don't know how to save the settings.
The only setting that works is enabling/disabling mouse logo

**Comment by nmasnadithya** [comment]
> > Everything works properly. Only issue I've seen is when I open RazerGenie the initial dpi values being weird. But it resolves automatically the moment I try to edit it
> 
> I'm trying to make it work but no success. Already cloned and builded the package from your branch. Device is detected, DPI values are weird. Cannot edit with Polychromatic. With RazerGenie, the values are weird but they get right after I try to edit. But still, they are not saving or I don't know how to save the settings.
> The only setting that works is enabling/disabling mouse logo

Thanks for the feedback. I thought it was something wrong with my RazerGenie installation. Will check this.

**Review by z3ntu — APPROVED** [review]
LGTM, thanks for the PR!

**Pull request merged**